.env.example

```
@@ -36,3 +36,10 @@ AUTH_EMAIL_MAX_ATTEMPTS=5
 AUTH_EMAIL_VERIFIED_TTL_SECONDS=1800
 AUTH_EMAIL_FROM_ADDRESS=
 AUTH_EMAIL_SUBJECT="[GACHI] Email verification code"
+AUTH_RATE_LIMIT_ENABLED=true
+AUTH_RATE_LIMIT_KEY_PREFIX=auth:rate-limit:
+AUTH_RATE_LIMIT_EMAIL_HMAC_SECRET=replace-with-strong-random-secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUTH_RATE_LIMIT_EMAIL_HMAC_SECRET key should go before the AUTH_RATE_LIMIT_ENABLED key

(UnorderedKey)